### PR TITLE
transitive list is expanded by default

### PR DESCRIPTION
**Pull request by chrisraygill**
Transitive header in Installed tab will be expanded by default.

### CODE REVIEW

File: proposed/2020/Transitive-Dependencies.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ Finally, selecting a transitive dependency will provide the user the ability to
 
 #### Detailed design
 
-**Collapsible, vertical list group:** Direct and transitive packages will be grouped under collapsible list group
+**Collapsible, vertical list group:** Direct and transitive packages will be grouped under collapsible list group (both expanded by default).
```
Comment (jebriede) [inline proposed/2020/Transitive-Dependencies.md:81]: Can you also update the following:
https://github.com/NuGet/Home/blob/d180ca7b610156a9acbc45f57ad777eaf4a78431/proposed/2020/Transitive-Dependencies.md?plain=1#L71

https://github.com/NuGet/Home/blob/d180ca7b610156a9acbc45f57ad777eaf4a78431/proposed/2020/Transitive-Dependencies.md?plain=1#L85

https://github.com/NuGet/Home/blob/d180ca7b610156a9acbc45f57ad777eaf4a78431/proposed/2020/Transitive-Dependencies.md?plain=1#L94

Comment (chrisraygill) [reply proposed/2020/Transitive-Dependencies.md:81]: Done 👍

**Review by jebriede — APPROVED** [review]

**Comment by nkolev92** [comment]
Team review: Does this need merged @chgill-MSFT

**Review by JonDouglas — APPROVED** [review]
merge?

**Pull request closed without merging**